### DISCUSSION TITLE
Better Name

### DISCUSSION THREAD

**Discussion by timvahlbrock** [Oeffis/Oeffis#1] [General]
We need a better, more recognisable name for the organisation, as well as the project.

**Comment by timvahlbrock** [comment]
Maybe something that brings the "intelligence" idea of the project more into focus? Something like
- Smarter Transit
- Transit Genius

**Comment by Paulomart** [comment]
_Öffis_ and `Oeffis`

**Reply by JaJako** [reply]
As an app name I like "Öffis"/"Oeffis" - its catchy and easy to remember. (At least for German users.)

**Comment by JaJako** [comment]
What about giving our organisation some fancy or just cool-sounding name, which does not necessarily need to have a direct correlation to our app. For example I found a company named "Super Flexible Software GmbH". 

Some ideas:
- Changing (Departure) Platform Software
- Highspeed Software Solutions
- Train(ed) Software Solutions
- Delayed Provision Software (Solutions)

**Comment by timvahlbrock** [comment]
For now, closed with result "Öffis"
